justfile

```diff
@@ -49,6 +49,11 @@ url-check:
 style:
   air format .
 
+# From https://jarl.etiennebacher.com/#installation
+# Lint R code for any potential issues
+lint:
```

Comment:
I didn't include this in `run-all` yet, unless you all agree to also installing Jarl. I'm fine with having it separate from `run-all` for now. Thoughts?